client/src/pages/customer-pages/FlightMap/flightmap.css

```diff
@@ -0,0 +1,23 @@
+@media screen and (max-width: 500px) {
```

Comment:
Here's another spot where some of the styles probably apply to all browser widths, and some others just to max-width: 500px